_posts/2020-07-10-removing-negative-words.markdown

```diff
@@ -96,3 +104,20 @@ We have listened to the community. We have seen the trends. And we're committed
 This is not an isolated initiative. OmbuLabs is committed to sustaining an open, diverse and inclusive work environment as a whole. This is one facet of that environment, and an important one.
 
 There are several arguments, human and technical, to move away from negative terminology. I hope this article helps you not only understand why we're doing this, but to advocate change in your projects and help us change as a community.
+
+## Sources and References
+
+1. [Developers Debate Deleting ‘Master’ and ‘Slave’ Code Terminology](https://insights.dice.com/2020/06/16/developers-debate-deleting-master-slave-code-terminology/) by Nick Kolakowski o[Tech Confronts Its Use of the Labels ‘Master’ and ‘Slave’n [Dice](https://insights.dice.com/).
+2. [There’s an industry that talks daily about ‘masters’ and ‘slaves.’ It needs to stop.](https://www.washingtonpost.com/opinions/2020/06/12/tech-industry-has-an-ugly-master-slave-problem/) by Sinclair Im on [The Washington Post](https://www.washingtonpost.com/)
```

Comment:
```suggestion
2. [There’s an industry that talks daily about ‘masters’ and ‘slaves.’ It needs to stop.](https://www.washingtonpost.com/opinions/2020/06/12/tech-industry-has-an-ugly-master-slave-problem/) by Sinclair Im in [The Washington Post](https://www.washingtonpost.com/)
```

Reply:
@FionaDL should this really be "in"? Just asking because I have been asked by a paper's reviewer to change to "on" when it's a website and not the physical magazine / newspaper / book.